packages/tas_keybind/init.lua

```diff
@@ -0,0 +1,79 @@
+tas = tas or {}
+tas._oldControls = {}
+
+---@alias tas.KeyListenerCallback fun(player: unknown, pressed: boolean):void
+---@alias Set<T> table<T, boolean>
+
+---@type table<tas.CONTROLS, Set<tas.KeyListenerCallback>>
+tas.keyListeners = {}
+---@enum tas.CONTROLS
+tas.CONTROLS = {
+	up = 0,
+	down = 1,
+	left = 2,
+	right = 3,
+	jump = 4,
+	aux1 = 5,
+	sneak = 6,
+	dig = 7,
+	place = 8,
+	zoom = 9,
+}
+
+local function saveControlsForPlayer(player)
+	local playerControls = player:get_player_control_bits()
```

Comment:
nit: Having `oldControls` and `playerControls` doesn't really distinguish between the two variables. Can we call this `newControls` here?

Reply:
ok